src/codeu/chat/database/Database.java

```diff
@@ -0,0 +1,91 @@
+package codeu.chat.database;
+
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import com.mongodb.MongoClientURI;
+import org.bson.Document;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+public class Database {
+    public MongoClient mongoClient;
```

Comment:
Good practice is to make everything as `private` as possible, and fields in particular should virtually never be public; an accessor method to get the instance is customary.

Likewise, mark every class and field `final` that you can. (This advice doesn't necessarily extend to local variables, where `final` doesn't matter *as much*, and a consistent usage of it can get noisy.)